castle/cms/static/patterns/focalpointselect.js

```diff
@@ -85,32 +86,34 @@ define([
 
     render: function(){
       var that = this;
-      if(that.state.hidden){
```

Comment:
Is this case being handled in some other way?

Reply:
Well the image doesn't actually get hidden anymore... I suppose maybe the state property should be renamed. Though I don't know if that could have unintentional consequences.

Reply:
Makes sense....I'd recommend just looking for any references to the hidden value, and verify it's not being used anywhere unexpected. Otherways, should be good

Reply:
I didn't see it anywhere else. Gave a more descriptive name.